### ISSUE TITLE
`$ Hexaville deploy <projectname>` fails with HexavilleCore.LauncherError

### ISSUE THREAD

**Issue by mitsuyoshi-yamazaki** [noppoMan/Hexaville#27]
# Agenda

`$ Hexaville deploy <projectname>` fails with `HexavilleCore.LauncherError`.

using:
Xcode 8.3.2
`Hexaville refs/tags/0.1.5`

# How to reproduce

Not sure. Under investigation.

# Error Logs

```shell
$ Hexaville deploy hoge
...
/Users/Mitsuyoshi/Documents/Development/Projects/Hoge/Sources/RandomNumberGenerateMiddleware.swift:13:9: error: value of type 'ApplicationContext' has no member 'storage'
        context.storage["randumNumber"] = randomNumber
        ^~~~~~~ ~~~~~~~
<unknown>:0: error: build had 1 command failures
error: exit(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f /Users/Mitsuyoshi/Documents/Development/Projects/Hoge/.build/debug.yaml
swiftBuildFailed
An error occurred: The operation couldn’t be completed. (HexavilleCore.LauncherError error 1.)
```

**Comment by mitsuyoshi-yamazaki** [comment]
seems like it fixed on master
https://github.com/noppoMan/Hexaville/blob/35d18160339339c2376c39dec0316feec15bd0e4/templates/SwiftProject/Sources/RandomNumberGenerateMiddleware.swift#L13

**Comment by noppoMan** [comment]
@mitsuyoshi-yamazaki 
I published version`0.1.6` that contains 35d18160339339c2376c39dec0316feec15bd0e4
